blocks/concat.py

```diff
@@ -53,3 +53,11 @@ class VectorConcat(Concat):
 
     def transform(self, data: List[List[int]]) -> pd.Series:
         return list(zip(*data))
+
+
+class ClassificationOutputConcat(Concat):
```

Comment:
a shortcut, so I can easily create a meta model, with a single concat instead of a transformation at the end of the pipeline + VectorConat